pkg/defaultRules/defaultRules.yaml

```diff
@@ -1869,3 +1869,58 @@ rules:
         $ref: "#"
       items:
         $ref: "#"
+  - id: 61
+    name: Ensure seccomp profile must not be explicitly set to Unconfined
```

Comment:
I think it will be more clear to the user if you will use "prevent" instead of "ensure ... not".
so maybe "Prevent setting 'Unconfined' as seccomp profile"?